mobly/controllers/attenuator_lib/telnet_client.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3.4
+#
+# Copyright 2016 Google Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Helper module for common telnet capability to communicate with
+AttenuatorDevice(s).
+
+User code shouldn't need to directly access this class.
+"""
+
+import telnetlib
+from mobly.controllers import attenuator
+
+
+def _ascii_string(uc_string):
+    return str(uc_string).encode('ASCII')
+
+
+class TelnetClient(object):
+    """This is an internal helper class for Telnet+SCPI command-based
```

Comment:
Is there anything specific to SCPI in this class? If so should it be telnet_scpi_client?